sdk/extendedlocation/Microsoft.Azure.Management.ExtendedLocation/tests/Microsoft.Azure.Management.ExtendedLocation.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props"/>
+  <PropertyGroup>
+    <PackageId>ExtendedLocation.Tests</PackageId>
+    <AssemblyName>ExtendedLocation.Tests</AssemblyName>
+    <Version>1.0.0</Version>
+    <Description>Test Project for Automation tests</Description>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.7.1"/>
+    <PackageReference Include="MSTest.TestAdapter" Version="2.1.1"/>
```

Comment:
You shouldn't use MSTest in these tests - you should use xunit

Reply:
we should remove these

Reply:
I have updated to not include these package references